docker/development/run.sh

```diff
@@ -20,7 +24,18 @@ _migrate() {
 }
 
 _octane() {
-  exec ./artisan octane:start --host=0.0.0.0 "$@"
+    echo_counter=0
+    while [ ! -f "$MANIFEST_PATH" ]; do
+        if [ "$echo_counter" -le 0 ]; then
+            echo_counter="$MANIFEST_MISSING_LOG_STEPS"
+            echo "waiting to start octane: $(basename "$MANIFEST_PATH") not found" >&2
```

Comment:
minor thing but there's no need to call external program just get basename
```suggestion
            echo "waiting to start octane: ${MANIFEST_PATH##*/} not found" >&2
```